alert-issuetracker/src/main/java/com/synopsys/integration/alert/issuetracker/message/IssueContentLengthValidator.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * alert-issuetracker
+ *
+ * Copyright (c) 2019 Synopsys, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.alert.issuetracker.message;
+
+import org.apache.commons.lang3.StringUtils;
+
+import com.synopsys.integration.alert.issuetracker.exception.IssueTrackerContentFormatException;
+
+public abstract class IssueContentLengthValidator {
+    public abstract int getTitleLength();
```

Comment:
Should this and the other methods be named `get...MaxLength()`? Should the methods be public?